### PR TITLE
Asaikovskiv2

### PR DESCRIPTION
**Pull request by AaronSaikovski**
added powershell script and markdown for Bicep

### CODE REVIEW

File: Bicep_Script_Create_WP/readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Trademarks
+
+This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft trademarks or logos is subject to and must follow [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
+Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
+Any use of third-party trademarks or logos are subject to those third-party's policies.
```
Comment (btardif) [inline Bicep_Script_Create_WP/readme.md:42]: This is unnecessary as it's already part of the base repo disclosure.

File: Bicep_Script_Create_WP/deploy.ps1 [hunk 2]
```diff
@@ -0,0 +1,37 @@
+##Deploy Biceo teplate using powershell
```
Comment (btardif) [inline Bicep_Script_Create_WP/deploy.ps1:1]: Typo "Biceo" should be "Bicep"?

File: Bicep_Script_Create_WP/deploy.ps1 [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+$resourceGroupName="wordpress-appsvc-rg"
+$location="West US"
+$bicepSrc="main.bicep"
+$armOutput="main.json"
+$templateParamFile="wp.dev.parameters.json"
+
+
+#Select Subscription
+# $subscriptionId = (Get-AzSubscription -SubscriptionName $subscriptionName).Id
+# Select-AzSubscription -SubscriptionId $subscriptionId
```
Comment (btardif) [inline Bicep_Script_Create_WP/deploy.ps1:17]: Should this be commented out?

File: Bicep_Script_Create_WP/deploy.ps1 [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#Deploy the Decompiled ARM template with param file
+New-AzResourceGroupDeployment `
+  -ResourceGroupName $resourceGroupName `
+  -TemplateFile $armOutput `
+  -TemplateParameterFile $templateParamFile ##-WhatIf
+  
+  ##-AsJob
+
+#Remove Resource Group
+##Remove-AzResourceGroup -Name $resourceGroupName -Force -AsJob
```
Comment (btardif) [inline Bicep_Script_Create_WP/deploy.ps1:35]: Dirty code? remove unnecessary comments?

File: Bicep_Script_Create_WP/deploy.ps1 [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+New-AzResourceGroup -Name $resourceGroupName -Location $location -Force
+
+#Convert Bicep to .JSON ARM template - issues with cmdlet - New-AzResourceGroupDeployment
+az bicep build --file $bicepSrc
+
+#Deploy the Decompiled ARM template with param file
+New-AzResourceGroupDeployment `
+  -ResourceGroupName $resourceGroupName `
+  -TemplateFile $armOutput `
+  -TemplateParameterFile $templateParamFile ##-WhatIf
```
Comment (btardif) [inline Bicep_Script_Create_WP/deploy.ps1:30]: This is a hack no? it's not really doing Bicep but rather converting Bicep to ARM and then executing ARM template

**Review by rabollin — APPROVED** [review]

**Review by AaronSaikovski — COMMENTED** [review]
done

**Pull request closed without merging**